collect_app/src/main/res/drawable/project_icon_background.xml

```diff
@@ -2,5 +2,5 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="oval">
     <solid android:color="@android:color/black"/>
-    <size android:width="30dp" android:height="30dp"/>
+    <size android:width="@dimen/project_list_icon_size" android:height="@dimen/project_list_icon_size"/>
```

Comment:
Personally I prefer just having these values inline as they're not going to be shared with anything else. Do you think they really need to go into `dimens.xml`?

Reply:
They're currently used as `marginStart` for the two settings buttons which is kind of weird. It's something I hope we come back to sooner than later but this might just be good enough that we leave it as-is. I thought adding the resource would be a useful breadcrumb for when we return to it.

Reply:
That feels like a "happens to be the same" rather than "is the same thing" to me, so I'd rather just inline it everywhere I think. The two buttons could anchor to a guideline instead of having individual margins on them as well. I'm know I'm being a bit of a stickler with all this, but these values just sit in a global namespace that's super hard to organize.

If you really think they need a shared value then I think the value should at least move to `collect_app`'s `dimens.xml` rather than have it in `androidshared`.

Reply:
I introduced the resource because it felt helpful to connect the icon size to the margin but if it's not, then let's just use constants. That also lets me do a margin of 38 which is icon_size + margin and better aligns. 

As far as I know, the best we could do with guidelines is introduce one at 38dp which I don't think is an improvement over the two margins. The project icon is in a separate layout which is what makes doing this dynamically a trickier thing.

Reply:
Yeah that seems good! Thanks for indulging me here.